nixos/modules/services/networking/bee-clef.nix

```diff
@@ -93,13 +96,16 @@ in {
         home = cfg.dataDir;
         createHome = false;
         isSystemUser = true;
-        description = "bee-clef user";
-        uid = config.ids.uids.bee-clef;
+        description = "user for the bee-clef service";
+        # TODO why does this fail for me? i see the exact same thing in other services
+        #uid = config.ids.uids.bee-clef;
```

Comment:
@SuperSandro2000 any comment on this?

do i need this? if yes, why doesn't it work?

i assume it's some kind of database to make uid and gid sticky if the users are coming and going according to the config changes?

Reply:
You don't need to use a static `uid` and `gid`. See https://github.com/NixOS/rfcs/blob/master/rfcs/0052-dynamic-ids.md for details.